dom/shopping-list.css

```diff
@@ -0,0 +1,11 @@
+shopping-list.css
+
+li {
+margin-bottom: 10px;
+}
+
+li button {
+font-size: 8px;
+margin-left: 20px;
+color: #666;
```

Comment:
Just be convention, any time you have a code block encapsulated by { }, you should tab over for everything in it. It beautifies your code and makes it more readable. (A shortcut to format your code automatically for this on VSCode is Alt+Shift+F)